enhancements/update/targeted-update-edge-blocking.md

```diff
@@ -0,0 +1,411 @@
+    If `clusters` is unset, the edge is blocked for all clusters.
+    * `promql` (optional, [object][json-object]), defining the subset of affected clusters, as described in [the blocked-edges section](#enhanced-graph-data-schema-for-blocking-edges).
+    * FIXME: [`platform` or other properties that are easier to evaluate than PromQL?](#non-promql-filters)
+
+### Enhanced ClusterVersion representation
+
+[The ClusterVersion `status`][api-cluster-version-status] will be extended with a new `conditionalUpdates` property:
+
+```go
+// conditionalUpdates contains the list of updates that are
+// conditionally recommended for this cluster. Consumers interested
+// in the set of updates that are actually recommended for this
+// cluster should use availableUpdates. This list may be empty if no
+// updates are recommended, if the update service is unavailable, or
+// if an empty or invalid channel has been specified.
+// +optional
+conditionalUpdates []ConditionalUpdate `json:"conditionalUpdates,omitempty"`
+```
+
+The `availableUpdates` documentation will be adjusted to read:
+
+```go
+// availableUpdates contains the subset of conditionalUpdates that
+// apply to this cluster.  Updates which appear in conditionalUpdates
+// but not in availableUpdates may expose this cluster to known
+// issues.
+```
+
+The new ConditionalUpdate type will have the following schema:
+
+```go
+// ConditionalUpdate represents an update which is recommended to some
+// clusters on the version the current cluster is reconciling, but which
+// may not be recommended for the current cluster.
+// +k8s:deepcopy-gen=true
+type ConditionalUpdate struct {
+	// release is the target of the update.
+	// +required
+	release Release `json:"release"`
+
+	// url contains information about this release. This URL is set by
+	// the 'url' metadata property on a release or the metadata returned by
+	// the update API and should be displayed as a link in user
+	// interfaces. The URL field may not be set for test or nightly
+	// releases.
+	// +optional
+	URL URL `json:"url,omitempty"`
+
+	// FIXME: [also reason/message?](#metadata-to-include-when-blocking-a-conditional-request)
+
+	// conditions represents the observations of the conditional update's
+	// current status. Known types are:
+	// * Recommended, for whether the update is recommended for the current cluster.
+	// +patchMergeKey=type
+	// +patchStrategy=merge
+	// +listType=map
+	// +listMapKey=type
+	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
+}
+```
+
+[ClusterVersion's `status.history` entries][api-history] will be extended with the following property:
+
+```go
+// overrides records update guards which were overriden to initiate the update.
+Overrides []string `json:"overrides,omitempty"`  FIXME: or maybe just a single string?  These will be populated via Go errors; not sure how structured we want this to be.
```

Comment:
Is this intended to contain any reason an admin used `force`, e.g. overrode a pending resource deletion, or just that one or more of these newly added conditions has been overridden? I think it should be the former, and the comment leads me to believe that as well, but it needs clarification either way.